src/main/java/nl/thehyve/ocdu/soap/ResponseHandlers/ImportDataResponseHandler.java

```diff
@@ -20,14 +20,24 @@
     /**
      * Checks if an error occurred on the OpenClinica-side and reports it back as the
      * return value
-     *
      * @param response the SOAP-response.
      * @return a non <code>null</code> error code.message if an error occurred. Some are reported by the OpenClinica-WS
      * instance at url. Returns <code>null</code> if everything went OK.
      * @throws Exception if a technical error occurs.
      */
     public static String parseImportDataResponse(SOAPMessage response) throws Exception {
-        return parseGenericResponse(response, ".//importDataResponse");
```

Comment:
Probably the easiest way to fix that will be to checkout e75fbab476f5199175614b734aa9ec8cd4125158 version of file ImportDataResponseHandler.java using:
 
        git checkout e75fbab476f5199175614b734aa9ec8cd4125158 PATH_TO_/ImportDataResponseHandler.java
and then add it to a new commit and push. I hope this is the only class modified in my last PR and in the commit you rebased on